tracer/src/Datadog.Trace/Processors/ObfuscatorTraceProcessor.cs

```diff
@@ -34,7 +61,7 @@ public Span Process(Span span)
         {
             if (span.Type == "sql" || span.Type == "cassandra")
             {
-                span.ResourceName = ObfuscateSqlResource(span.ResourceName);
+                span.ResourceName = SqlObfuscator(span.ResourceName);
```

Comment:
nitpicking again - (and clearly don't mind me) 
But as it's not a class, but a function I change that to a verb. So `ObfuscateSql` or `ObfuscateSqlResource` would be better IMO

Reply:
TIL, yeah that does make, sense, made the new commits, will wait for the checks to run so I can merge.